docs/Community/sig-architecture/OWNERS

```diff
@@ -1,5 +1,5 @@
 chair:
-  - CodeMonkeyLeet
+  - MWShan [Proposed]
```

Comment:
@MWShan I'm suggesting that you take over the chair for sig-arch, at least for the near future since so much of the upcoming work will be around EDMM support. No pressure, feel free to nominate others as well.